src/components/Contentful/DynamicHeroClassic.vue

```diff
@@ -11,7 +11,7 @@
 					:style="customGridStyles"
 				>
 					<div
-						class="tw-mx-auto tw-col-span-12 md:tw-col-span-6 xl:tw-col-span-5"
+						class="tw-mx-auto tw-col-span-12 md:tw-col-span-6"
```

Comment:
Did you consult with Design on this change? I think this will now change the expected layout for the homepage hero instances...